docs/danews.md

```diff
@@ -0,0 +1,147 @@
+The dataset consist of 25874862 documents where 1048815 (0.04%) are left after filtering. 
+
+**Does the dataset contain all possible instances or is it a sample (not necessarily random) of instances from a larger set?**
+Prior to filtering this dataset contains all instances of new articles from the given period across the sources.
+
+**If the dataset is a sample from a larger set, what was the sampling strategy?**
+The dataset is not a sample, however it is filtered as described in post processing.
+
+**Who was involved in the data collection process?**
+A team of researchers at Center for Humanities
+Computing Aarhus (CHCAA) recieved this dataset in a collaboration with Infomedia.
+
+**Over what timeframe was the data collected?**
+The dataset include articles from the period 01/12/2000 to  30/04/2021.
+
+**Were any ethical review processes conducted?**
+No.
+
+## Preprocessing/cleaning/labeling
+
+**Was any preprocessing/Cleaning/Labeling of the data done 
+(e.g., discretization or bucketing, tokenization, part-of-speech tagging, 
+SIFT feature extraction, removal of instances, processing of missing values)?**
+
+We filter documents that are not in Danish. We remove low-quality text and deduplicate documents based on their 
+document-level n-gram similarity. The full pre-processing details are given in the post processing section.
+
+**Is the software used to preprocess/clean/label the instances available?**
+Yes, the script are avaiable [here](https://github.com/centre-for-humanities-computing/danish-foundation-models/tree/main/src/application/danews). 
+Which used version 0.0.1 of the [dfm package](https://github.com/centre-for-humanities-computing/danish-foundation-models). 
+
+## Uses
+**Has the dataset been used for any tasks already?**
+Yes, the dataset has been used to pre-train Danish language models.
+Parts of the dataset have also been used in
+Baglini, R. B., Nielbo, K. L., Hæstrup, F., Enevoldsen, K., Vahlstrup, P. B., & Roepstorff, A. (2021, June 2). When no news is bad news: Detection of negative events from news media content. https://2021.dhbenelux.org/
+Nielbo, K. L., Baglini, R. B., Vahlstrup, P. B., Enevoldsen, K. C., Bechmann, A., & Roepstorff, A. (2021, January). News information decoupling: An information signature of catastrophes in legacy news media. https://eadh2020-2021.org/
+
+**Is there a repository that links to any or all papers or systems that use the dataset?**
+No.
+
+**What (other) tasks could the dataset be used for?**
+The scale of the dataset makes it suitable for NLP tasks such as language modelling. Similarly, the structure
+of the articles make it a suitable dataset for training text-summerization models.
+
+**Is there anything about the composition of the dataset or the way it was collected and preprocessed/cleaned/labeled that might impact future uses?**
+
+This dataset is static and thus does not evolve over time with the language, thus will become increasingly outdated over time.
```

Comment:
```suggestion
This dataset is static and thus does not evolve over time with the language. A consequence of this is that it will become increasingly outdated over time.
```

Reply:
updated in datasheet